### PR TITLE
[Item Identification] Add sacks

### PR DESCRIPTION
**Pull request by Silverfeelin**
Fixes https://github.com/runelite/runelite/issues/9371 by adding identifiers to sacks.

Empty, EMP
Onion, ONI
Cabbage, CAB
Potato, POT

image
image

image

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationConfig.java [hunk 1]
```diff
@@ -64,6 +64,16 @@ default boolean showSeeds()
...
 	}
 
+	@ConfigItem(
+			keyName = "showSacks",
+			name = "Sacks",
+			description = "Show identification on Sacks"
+	)
+	default boolean showSacks()
+	{
+		return true;
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationConfig.java]: This should default to `false`.

**Comment by Silverfeelin** [comment]
Done. I figured true would make more sense because they're unidentifiable without the item name.

**Comment by Nightfirecat** [comment]
The general sentiment lately has been to try to keep more new features (particularly more fringe features) defaulted off when added.

File: runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationConfig.java [hunk 2]
```diff
@@ -64,6 +64,16 @@ default boolean showSeeds()
 		return true;
 	}
 
+	@ConfigItem(
+			keyName = "showSacks",
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationConfig.java]: Fix your indentation (continuation indent 8>4)

Comment (Silverfeelin) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationConfig.java]: For some reason IntelliJ defaults to this indenting when starting a new line. Visual Studio and VS Code have spoiled me too much. Either way, fixed.

Comment (deathbeam) [reply runelite-client/src/main/java/net/runelite/client/plugins/itemidentification/ItemIdentificationConfig.java]: Yeah, I know it defaults to that. We have wiki page here with instructions on how to import RL code style to intellij: https://github.com/runelite/runelite/wiki

**Pull request merged**